lib/prometheus/client/histogram.rb

```diff
@@ -94,6 +94,23 @@ def values
         end
       end
 
+      def init_label_set(labels)
+        base_label_set = label_set_for(labels)
+
+        @store.synchronize do
+          [*@buckets, "+Inf"].each do |bucket|
+            # This is basically faster than doing `.merge`
+            bucket_label_set = base_label_set.dup
+            bucket_label_set[:le] = bucket.to_s
+            sum_label_set = base_label_set.dup
+            sum_label_set[:le] = "sum"
+
+            @store.set(labels: bucket_label_set, val: 0)
+            @store.set(labels: sum_label_set, val: 0)
+          end
+        end
+      end
```

Comment:
I just realised that this client does not provide a `_count` metric as part of the histogram. Is this true? Or did I miss it?

Reply:
Hrmm, good question, there should be a count, it's required for histograms. Maybe it's auto-generated from the `+Inf` bucket (they should be the same value).

Reply:
@jbernardo95 The client does provide a count in the export to prometheus, it just doesn't need to explicitly store it in the hash, because as SuperQ said, it's the same as the +Inf value.

This is where the magic happens: https://github.com/prometheus/client_ruby/blob/master/lib/prometheus/client/formats/text.rb#L67

Reply:
Makes sense 👍